changelogs/1.23.0.yaml

```diff
@@ -63,6 +63,9 @@ new_features:
 - area: ext_proc
   change: |
     added support for per-route :ref:`grpc_service <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExtProcOverrides.grpc_service>`.
+- area:  http2
+  change: |
+    added initial implementation for deferring processing of backed up http2 streams by buffering in the receiving codec. This is guarded by ``envoy.reloadable_features.defer_processing_backedup_streams``, but is disabled by default. Trailers and bodies are buffered if a stream is backed up. Body data is processed in chunks up to the size of the receiving codec's buffer high watermark. Fairness is implemented by prioritizing streams that haven't written to the connection recently to process deferred data when we transition from network connection backed up to network connection low watermark. Defer processing applies to both upstream and downstream connections.
```

Comment:
optional: I'd be inclined to swap
 This is guarded by ``envoy.reloadable_features.defer_processing_backedup_streams``, but is disabled by default.
This is off by default but can be enabled by setting ``envoy.reloadable_features.defer_processing_backedup_streams`` to true

Defer processing  -> Deferred processing?

though really given it's off by default I think we can just skip the release note